MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) allows user to select a Single date, Multiple dates or a Range of dates. It provides Month, Year, Decade, and Century view options for quickly navigate to the desired date. It supports minimum dates, maximum dates, enable past dates, and disabled dates to restrict the date selection.
+
+## Key features
+
+* **Multiple calendar view modes**: For selection and navigation, use Month, Year, Decade, and Century calendar views.
+
+   {% multiple_column split:2 %}
+   {% column %}
+   ![Month view in .NET MAUI Calendar.](images/overview/maui-month-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Year view in .NET MAUI Calendar.](images/overview/maui-year-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Decade view in .NET MAUI Calendar.](images/overview/maui-decade-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Century view in .NET MAUI Calendar.](images/overview/maui-century-view.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the desired date using different calendar views in the SfCalendar.
```

Comment:
You can easily navigate to the desired date using different calendar views in the `SfCalendar.`

Reply:
Changed the content